addons/web/static/src/js/views/basic/basic_renderer.js

```diff
@@ -622,10 +623,12 @@ var BasicRenderer = AbstractRenderer.extend({
         var modifiers = this._registerModifiers(node, record, null, options);
         // Initialize and register the widget
         // Readonly status is known as the modifiers have just been registered
+        const nodeOptions = node.attrs.options && pyUtils.py_eval(node.attrs.options);
```

Comment:
seems ok to do this here, but could you check if we can remove the other py_eval of the options (in basic_view?)